docs/index.rst

```diff
@@ -1,16 +1,77 @@
+.. _baseband_docs:
+
 ********
 Baseband
 ********
 
-Welcome to the Baseband documentation! Baseband is a package for reading and 
+Welcome to the Baseband documentation! Baseband is a package for reading and
 writing VLBI and other radio baseband files, with the aim of simplifying and
-streamlining data conversion and standardization.
+streamlining data conversion and standardization.  It provides:
+
+- File input/output functions for supported radio baseband formats, listed under
```

Comment:
Critiqing my own writing, it seems silly to list "data containers" as a code feature, since the user doesn't directly access them, and will likely be manipulating decoded arrays and not, say, frame words.  Perhaps unify these points to:
```
- File input/output objects for supported radio baseband formats, enabling
selective decoding of data into Numpy `arrays <numpy.ndarray>`, and encoding
baseband formats from user-defined arrays. Supported formats are listed under
:ref:`specific file formats <specific_file_formats_toc>`.
```